pkg/channel/distributed/dispatcher/dispatcher/handler.go

```diff
@@ -6,42 +6,34 @@ import (
 	"github.com/Shopify/sarama"
 	kafkasaramaprotocol "github.com/cloudevents/sdk-go/protocol/kafka_sarama/v2"
 	"github.com/cloudevents/sdk-go/v2/binding"
-	"github.com/slok/goresilience/retry"
 	"go.uber.org/zap"
 	eventingduck "knative.dev/eventing/pkg/apis/duck/v1"
 	"knative.dev/eventing/pkg/channel"
-	"math"
+	"knative.dev/eventing/pkg/kncloudevents"
+	"net/http"
 	"net/url"
 	"regexp"
-	"strconv"
-	"time"
 )
 
 // 3 Digit Word Boundary HTTP Status Code Regular Expression
-var HttpStatusCodeRegExp = regexp.MustCompile(`(^|\s)([12345]\d{2})(\s|$)`)
+var HttpStatusCodeRegExp = regexp.MustCompile("(^|\\s)([12345]\\d{2})(\\s|$)")
```

Comment:
The linter will complain about this:
```
 Error: S1007: should use raw string (`...`) with regexp.MustCompile to avoid having to escape twice (gosimple)
```

Reply:
it did - i fixed it - thanks